content/references/rippled-api/api-conventions/request-formatting.md

```diff
@@ -1,70 +1,128 @@
+            "ledger_index": "validated",
+            "api_version": 1
         }
     ]
 }
 ```
 
 *Commandline*
 
-```
-rippled account_info r9cZA1mLK5R5Am25ArfXFmqgNwjZgnfk59 validated true
+```sh
+rippled account_info r9cZA1mLK5R5Am25ArfXFmqgNwjZgnfk59 validated strict
 ```
 
 <!-- MULTICODE_BLOCK_END -->
+
+
+## WebSocket Format  
+
+After you open a WebSocket to the `rippled` server, you can send commands as a [JSON](https://en.wikipedia.org/wiki/JSON) object with the following fields:
+
+| Field               | Type      | Description                                |
+|:--------------------|:----------|:-------------------------------------------|
+| `command`           | String    | The name of the [API method](public-rippled-methods.html). |
+| `id`                | (Various) | _(Optional)_ A unique value to identify this request. The response to this request uses the same `id` field. This way, even if responses arrive out of order, you know which request prompted which response. |
+| `api_version`       | Number    | _(Optional)_ The API version to use. If omitted, use version 1. For details, see [API Versioning](#api-versioning). [New in: rippled 1.5.0][] |
+| (Method Parameters) | (Various) | Provide any parameters to the method at the top level. |
+
+See [Response Formatting](response-formatting.html) for the response from the server.
+
+## JSON-RPC Format
+
+To make a JSON-RPC request, send an HTTP **POST** request to the root path (`/`) on the port and IP where the `rippled` server is listening for JSON-RPC connections. You can use HTTP/1.0 or HTTP/1.1. If you use HTTPS, you should use TLS v1.2. For security reasons, `rippled` _does not support_ SSL v3 or earlier.
+
+Always include a `Content-Type` header with the value `application/json`.
+
+If you plan on making multiple requests, use [Keep-Alives](http://tools.ietf.org/html/rfc7230#section-6.3) so that you do not have to close and re-open the connection in between requests.
+
+Send request body as a [JSON](https://en.wikipedia.org/wiki/JSON) object with the following fields:
+
+
+| Field               | Type      | Description                                |
+|:--------------------|:----------|:-------------------------------------------|
+| `method`            | String    | The name of the [API method](public-rippled-methods.html). |
+| `params`            | Array     | _(Optional)_ A **one-item array** containing a nested JSON object with the parameters to this method. You may omit this field if the method does not require any parameters. |
+
+The object inside the `params` array can contain the following fields:
+
+| Field               | Type      | Description                                |
+|:--------------------|:----------|:-------------------------------------------|
+| `api_version`       | Number    | _(Optional)_ The API version to use. If omitted, use version 1. For details, see [API Versioning](#api-versioning). [New in: rippled 1.5.0][] |
```

Comment:
In "[New in: rippled 1.5.0][]", the "[]" will be filled later, right?

Reply:
It'll get filled out on the live site, yes. [The included `rippled_versions.md` file](https://raw.githubusercontent.com/ripple/xrpl-dev-portal/master/content/_snippets/rippled_versions.md) defines this link, among others.

Reply:
Thanks.